tests/test_main.py

```diff
@@ -157,8 +141,8 @@ def test_get_block_coverage_noresults_live(auth_live, monkeypatch):
     # pylint: disable=unused-variable
     try:
         block_id = "045019bb-06fc-4fa1-b703-318725b4d8af"
-        coverage = get_block_coverage(block_id=block_id)
-    except requests.exceptions.RequestException as err:
+        _ = get_block_coverage(block_id=block_id)
+    except requests.exceptions.RequestException:
         assert True
         return
     assert False
```

Comment:
I will update this, but why are we spending any energy at all for the tests on blocks? 😅

Reply:
1. Opportunity to educate via review comments because similar problems [might] appear in other places, most of the good patterns need to rely on muscle memory rather than rational discourse during work
2. Uniform consistency to simplify maintenance if needed
3. They are already included in the PR